protocol/contracts/beanstalk/init/InitDistribution.sol

```diff
@@ -0,0 +1,49 @@
+/*
+ SPDX-License-Identifier: MIT
+*/
+
+pragma solidity ^0.8.20;
+
+import {AppStorage} from "contracts/beanstalk/AppStorage.sol";
+import {C} from "contracts/C.sol";
+import {LibDiamond} from "contracts/libraries/LibDiamond.sol";
+import {Storage} from "contracts/beanstalk/AppStorage.sol";
+
+// NOTE: Values are arbitrary placeholders.
+
+/**
+ * @author funderbrker
+ * @notice Initializes logistics and shipment routes.
+ */
+contract InitDistribution {
+    AppStorage internal s;
+
+    function init() external {
+        s.shipmentRoutes.push(
```

Comment:
This works as an init, but ideally we should probably have functionality somewhere that allows people to add, modify, and remove shipment routes. We should also have events that correspond to these state changes. In my head, this looks pretty similar to how LibWhitelistedTokens.sol is set up. Could we implment a library that does that and use that here?

Reply:
Definitely.

Thankfully this system of updating can be much simpler than LibWhitelisted tokens.

Added here https://github.com/BeanstalkFarms/Beanstalk/pull/860/commits/08614d4814895da7c59ddd8c24b0eb489ca783f0#diff-a3ff8fea8dfdc60875a144d7d0f1fb3b1e5f0b82f2cbcb1331f2fca38d73f833R120

Reply:
Beautiful! works for me!